dotcom-rendering/src/components/FootballTablesPage.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from '@storybook/react/*';
+import { fn } from '@storybook/test';
+import { regions } from '../../fixtures/manual/footballData';
+import { Default as TableDefault } from './FootballTable.stories';
+import { FootballTablesPage } from './FootballTablesPage';
+
+const meta = {
+	title: 'Components/Football Tables Page',
+	component: FootballTablesPage,
+} satisfies Meta<typeof FootballTablesPage>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
+	args: {
+		regions,
+		goToCompetitionSpecificPage: fn(),
+		pageId: 'football/tables',
+		tables: [
+			{ ...TableDefault.args.table },
+			{ ...TableDefault.args.table },
+		],
```

Comment:
As above, 👍 for story reuse!

The object could probably be used directly[^1], rather than spread, or you could reuse the stories for `FootballTableList`?

```suggestion
		tables: FootballTableList.args.tables,
```

[^1]: https://storybook.js.org/docs/writing-stories/build-pages-with-storybook#args-composition-for-presentational-screens